code/datums/components/chewable.dm

```diff
@@ -0,0 +1,80 @@
+/// Anything with this component will provide the reagents inside the
+/// item to the user when it is equipped.
+/datum/component/chewable
+	/// A bitfield of valid slots. If this is not provided, then it will
+	/// use the `slot_flags` of the item.
+	var/slots_to_check
+
+	/// The time left before it's deleted.
+	var/lifetime = 6 MINUTES
+
+	/// The interval to give reagents while equipped.
```

Comment:
just give less reagents per process

Reply:
Same as above, this was done this way on cigarettes because transferring very low amount of chems would make whatever you're transferring to a mob have no effect whatsoever, this needs to be tested if it's still the case if you're going to change it back